molecule/certificates.yml

```diff
@@ -20,6 +20,31 @@
         state: present
       when: ansible_os_family == "Debian"
 
+    - name: Get Java Path
```

Comment:
Do you want to add a check for fips_enabled flag on non RedHat system.

Reply:
As per official CP docs, we don't support fips on non redhat systems

Reply:
will add a task to fail the setup when fips is enabled on non redhat